server/src/parse/tests.rs

```diff
@@ -0,0 +1,316 @@
+/// Unittest for parsing module
+
+
+use std::iter::Iterator;
+use super::ast::*;
+use super::token:: {TokenSpan, Lit};
+use super::lex::Lexer;
+use std::mem::swap;
+use super::token::Token;
+use super::Span;
+use super::super::storage::SqlType;
+use super::parser;
+use std::collections::HashMap;
+
+// ============================================================================
+// Result::Ok unittest
+// ============================================================================
+
+#[test]
+fn test_create_table_empty() {
+    let mut p = parser::Parser::create("cReAtE table
+        foo");
```

Comment:
You know... you can put `\n` into a Rust string literal? That way it's more clear what you want to test ;)
Currently it looks like an accidental line break ^_^